File: VB-Examples/GetBarcodeInfo/GetBarcodeInfo.vb
Imports System.Text

Imports Spire.Barcode
Imports Spire.Barcode.Settings
Imports System.IO

Namespace GetBarcodeInfo
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()

		End Sub
		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click

			' Load barcode image
			Dim barcodeInfos() As BarcodeInfo = BarcodeScanner.ScanInfo("..\..\..\..\..\Data\GetBarcodeInfo.png")
			Dim stringBuilder As New StringBuilder()

			Dim point As String = ""
			Dim dataString As String = ""
			Dim vertexes() As Point = Nothing
			Dim barCodeReadType As BarCodeReadType = 0

			' Get barcode information
			For Each barcodeInfo As BarcodeInfo In barcodeInfos
				barCodeReadType = barcodeInfo.BarCodeReadType
				vertexes = barcodeInfo.Vertexes
				dataString = barcodeInfo.DataString

			Next barcodeInfo
			' Get vertexes
			For i As Integer = 0 To vertexes.Length - 1
				point &= vertexes(i).ToString()

			Next i

			'Add barcode information string to StringBuilder
			stringBuilder.AppendLine("barCodeReadType：" & barCodeReadType.ToString())
			stringBuilder.AppendLine("vertexes：" & point)
			stringBuilder.AppendLine("dataString：" & dataString)
			Dim result As String = "BarcodeInfo.txt"

			'Save them to a txt file
			File.WriteAllText(result, stringBuilder.ToString())

			'Launching the txt file.
			TextViewer(result)
		End Sub
		Private Sub TextViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

	End Class
End Namespace

File: CS-Examples/GetBarcodeInfo/GetBarcodeInfo.cs
using System;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

using Spire.Barcode;
using Spire.Barcode.Settings;
using System.IO;

namespace GetBarcodeInfo
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();

        }
        private void button1_Click(object sender, EventArgs e)
        {

            // Load barcode image
            BarcodeInfo[] barcodeInfos = BarcodeScanner.ScanInfo(@"..\..\..\..\..\Data\GetBarcodeInfo.png");
            StringBuilder stringBuilder = new StringBuilder();

            string point = "";
            string dataString = "";
            Point[] vertexes = null;
            BarCodeReadType barCodeReadType = 0;

            // Get barcode information
            foreach (BarcodeInfo barcodeInfo in barcodeInfos)
            {
                barCodeReadType = barcodeInfo.BarCodeReadType;
                vertexes = barcodeInfo.Vertexes;
                dataString = barcodeInfo.DataString;

            }
            // Get vertexes
            for (int i = 0; i < vertexes.Length; i++)
            {
                point += vertexes[i].ToString();

            }

            //Add barcode information string to StringBuilder
            stringBuilder.AppendLine("barCodeReadType：" + barCodeReadType.ToString());
            stringBuilder.AppendLine("vertexes：" + point);
            stringBuilder.AppendLine("dataString：" + dataString);
            String result = "BarcodeInfo.txt";

            //Save them to a txt file
            File.WriteAllText(result, stringBuilder.ToString());

            //Launching the txt file.
            TextViewer(result);
        }
        private void TextViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

    }
}
